### PR TITLE
bpo-35393: Fix typo

### PR DESCRIPTION
**Pull request by AutomCoding**
There are 256 characters in the range 0&ndash;255.

https://bugs.python.org/issue35393

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Doc/howto/unicode.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ Some were true standards, defined by the International Organization for
 Standardization, and some were *de facto* conventions that were invented by one
 company or another and managed to catch on.
 
-255 characters aren't very many.  For example, you can't fit both the accented
+256 characters aren't very many.  For example, you can't fit both the accented
```
Comment (ericvsmith) [inline Doc/howto/unicode.rst:52]: I can't leave a comment on line 44, but I think we might want to also change it to say: 
```
hold 256 unique values ranging from 0 to 255.  ASCII codes only went up to 127, so some
```
I'm concerned that a beginner will see 256 for the first time and wonder where it comes from.

Comment (terryjreedy) [reply Doc/howto/unicode.rst:52]: I agree, but suggest adding 'one of' to get 'bytes could hold one of 256 unique values ...'.

Comment (ericvsmith) [reply Doc/howto/unicode.rst:52]: Agreed that that's an improvement.

**Comment by csabella** [comment]
This doc page was rewritten as part of bpo-20906, so I am closing this issue as out of date.

**Pull request closed without merging**